src/components/Footer.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { FooterProps } from '../types/Props';
+
+export const Footer: React.FC<FooterProps> = ({ todos, filter, setFilter }) => {
+  const activeTodos = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodos} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link ${filter === 'all' ? 'selected' : ''}`}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilter('all')}
+        >
+          All
```

Comment:
As i wrote above, create an enum and use it here

Reply:
> As i wrote above, create an enum and use it here

Good point
You can convert your enum into an array with `Object.keys/values/entries` and map through it to generate links